src/OpenTelemetry.Api/.publicApi/netstandard2.0/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,327 @@
+static OpenTelemetry.Context.RuntimeContext.ContextSlotType.set -> void
+static OpenTelemetry.Context.RuntimeContext.GetSlot<T>(string slotName) -> OpenTelemetry.Context.RuntimeContextSlot<T>
+static OpenTelemetry.Context.RuntimeContext.GetValue<T>(string name) -> T
+static OpenTelemetry.Context.RuntimeContext.RegisterSlot<T>(string slotName) -> OpenTelemetry.Context.RuntimeContextSlot<T>
+static OpenTelemetry.Context.RuntimeContext.SetValue<T>(string name, T value) -> void
+static OpenTelemetry.Metrics.MeterFactoryBase.Default.get -> OpenTelemetry.Metrics.MeterFactoryBase
+static OpenTelemetry.Metrics.MeterFactoryBase.SetDefault(OpenTelemetry.Metrics.MeterFactoryBase meterFactory) -> void
+static OpenTelemetry.Metrics.MeterProvider.Default.get -> OpenTelemetry.Metrics.MeterProvider
+static OpenTelemetry.Metrics.MeterProvider.SetDefault(OpenTelemetry.Metrics.MeterProvider meterProvider) -> void
+static OpenTelemetry.Trace.ActivityExtensions.EnumerateEvents<T>(this System.Diagnostics.Activity activity, ref T eventEnumerator) -> void
+static OpenTelemetry.Trace.ActivityExtensions.EnumerateLinks<T>(this System.Diagnostics.Activity activity, ref T linkEnumerator) -> void
+static OpenTelemetry.Trace.ActivityExtensions.EnumerateTags<T>(this System.Diagnostics.Activity activity, ref T tagEnumerator) -> void
+static OpenTelemetry.Trace.ActivityExtensions.EnumerateTags<T>(this System.Diagnostics.ActivityEvent activityEvent, ref T tagEnumerator) -> void
+static OpenTelemetry.Trace.ActivityExtensions.EnumerateTags<T>(this System.Diagnostics.ActivityLink activityLink, ref T tagEnumerator) -> void
+static OpenTelemetry.Trace.ActivityExtensions.GetStatus(this System.Diagnostics.Activity activity) -> OpenTelemetry.Trace.Status
+static OpenTelemetry.Trace.ActivityExtensions.GetTagValue(this System.Diagnostics.Activity activity, string tagName) -> object
+static OpenTelemetry.Trace.ActivityExtensions.RecordException(this System.Diagnostics.Activity activity, System.Exception ex) -> void
+static OpenTelemetry.Trace.ActivityExtensions.SetStatus(this System.Diagnostics.Activity activity, OpenTelemetry.Trace.Status status) -> void
+static OpenTelemetry.Trace.Link.operator !=(OpenTelemetry.Trace.Link link1, OpenTelemetry.Trace.Link link2) -> bool
+static OpenTelemetry.Trace.Link.operator ==(OpenTelemetry.Trace.Link link1, OpenTelemetry.Trace.Link link2) -> bool
+static OpenTelemetry.Trace.SpanContext.implicit operator System.Diagnostics.ActivityContext(OpenTelemetry.Trace.SpanContext spanContext) -> System.Diagnostics.ActivityContext
+static OpenTelemetry.Trace.SpanContext.operator !=(OpenTelemetry.Trace.SpanContext spanContext1, OpenTelemetry.Trace.SpanContext spanContext2) -> bool
+static OpenTelemetry.Trace.SpanContext.operator ==(OpenTelemetry.Trace.SpanContext spanContext1, OpenTelemetry.Trace.SpanContext spanContext2) -> bool
+static OpenTelemetry.Trace.SpanHelper.ResolveSpanStatusForGrpcStatusCode(int statusCode) -> OpenTelemetry.Trace.Status
```

Comment:
let's not mention grpc in API surface. Even though the list may have been inherited from grpc in the first place